filters.js

```diff
@@ -447,6 +474,54 @@ function operationOnDates(v, arg, operation) {
   }
 }
 
+function toCurrency(currValue, currType) {
+  if (
+    isValidNumber(currValue) &&
+    _.isString(currType)
+  ) {
+    return { value: currValue, type: currType };
+  }
+  throw new Error("invalid currency value or type");
+}
+
+function toDuration(durValue, durType) {
+  if (
+    isValidNumber(durValue) &&
+    _.isString(durType)
+  ) {
+    const durationType = durType.toUpperCase();
+
+    switch (durationType) {
```

Comment:
Why do we need a switch case here? We are doing the exact same thing in every case. Let's just use one if condition for this.

Reply:
@sanket0896 have updated this now. I did not completely understand if I had to add a new if statement.